app.R

```diff
@@ -0,0 +1,1123 @@
+library(shiny)
+#library(png)
+library(shinyBS)
+#library(shinyDND)
+#library(shinyjs)
+library(ggplot2)
+library(dplyr)
+library(shinydashboard)
+library(simstudy)
+#library(lubridate)
+#library(shinyalert)
+library(shinyWidgets)
+library(boastUtils)
+library(fontawesome)
+
+
+convertMenuItem <- function(mi,tabName) {
+  mi$children[[1]]$attribs['data-toggle']="tab"
+  mi$children[[1]]$attribs['data-value'] = tabName
+  if(length(mi$attribs$class)>0 && mi$attribs$class=="treeview"){
+    mi$attribs$class=NULL
+  }
+  mi
+}
+maxScore <- 10
+
+ui <- list(
+  dashboardPage(
+    skin = "black", 
+    dashboardHeader(title = "ANCOVA",
+                    titleWidth = 250,
+                    tags$li(
+                      class = "dropdown",
+                      actionLink("info",
+                                 icon("info"))
+                    ),
+                    tags$li(
+                      class = "dropdown",
+                      tags$a(target = "_blank", 
+                             icon("comments"),
+                             href = "https://pennstate.qualtrics.com/jfe/form/SV_7TLIkFtJEJ7fEPz?appName=ANCOVA"
+                      )
+                    ),
+                    tags$li(
+                      class = "dropdown",
+                      tags$a(icon("home"),
+                             href = 'https://shinyapps.science.psu.edu/'
+                      )
+                    )
+    ),
+    #adding pages to sidebar
+    dashboardSidebar(
+      width = 220,
+      sidebarMenu(id = "pages",
+                  menuItem(text = "Overview",
+                           tabName = "instruction", 
+                           icon = icon("dashboard")
+                  ),
+                  menuItem(text = "Prerequisites", 
+                           tabName = "prereq", 
+                           icon = icon("book")
+                  ),
+                  menuItem(text = "Explore",
+                           tabName = "exploring", 
+                           icon = icon("wpexplorer")
+                  ),
+                  menuItem(text = "Game", 
+                           tabName = "game", 
+                           icon = icon("gamepad")
+                  ),
+                  menuItem(text = "References",
+                           tabName = "refs",
+                           icon = icon("leanpub")
+                  )
+      ),
+      tags$div(
+        class = "sidebar-logo", 
+        boastUtils::sidebarFooter()
+      )
+    ),
+    dashboardBody(
+      tabItems(
+        tabItem(tabName = "instruction",
+                h1("ANCOVA"),
+                h2("About:"),
+                p('This app introduces the concept of ANCOVA focusing on 
+                         interpreting interaction plots.'),
+                br(),
+                h2('Instructions:'),
+                p(
```

Comment:
For lines 90 - 116, instead of doing p() and the tags$li() try: 
tags$ul(
     tags$li(""), 
     tags$li(""), 
     tags$li(""), 
)
This way the bullet points won't look like they are so are so far to the left.